icu4c/source/i18n/nounit.cpp

```diff
@@ -11,7 +11,7 @@ U_NAMESPACE_BEGIN
 UOBJECT_DEFINE_RTTI_IMPLEMENTATION(NoUnit)
 
 NoUnit U_EXPORT2 NoUnit::base() {
-    return NoUnit("base");
+    return NoUnit("one");
```

Comment:
"one" is the numerator in "one-per-second", for example, which is where this identifier is used.  It would be weird to say "none-per-second" or "noUnit-per-second".  Ultimately the naming here is just pulling hairs, and I'm happy with `"one"` and don't see a strong reason to reconsider it again.